meta/parse.pl

```diff
@@ -263,6 +265,8 @@ sub ExtractDescription
 sai_qos_map_list_t      QOS_MAP_LIST
 sai_tunnel_map_list_t   TUNNEL_MAP_LIST
 sai_acl_capability_t    ACL_CAPABILITY
+sai_tlv_list_t          TLV_LIST
+sai_segment_list_t      SEGMENT_LIST
```

Comment:
According attr value types needs to be included i side saimrtadatsttpes.h And saisanitycheck.c needs to be updated